com.github.bytezz.IPLookup.json

```diff
@@ -0,0 +1,42 @@
+{
+    "app-id" : "com.github.bytezz.IPLookup",
```

Comment:
it should be `io.github`

Reply:
Changing that I would have to change a lot of things. Opened a [PR](https://github.com/flathub/flatpak-builder-lint/pull/139) for exception as suggested [here](https://github.com/flathub/flatpak-builder-lint/issues/15#issuecomment-1278743255).

Next project I start will be "io.github".